relocation.bs

```diff
@@ -698,17 +714,17 @@ constructed from the source object. The temporary is then destructed at the end
 of the expression evaluation.
 
 However, materializing a temporary whose only goal is to be destroyed is 
-suboptimal. Hence, it is authorized for implementations to elide the creation
-of the temporary object (if created by relocation or move constructor), 
+suboptimal. Hence, it is authorized for implementations to (as above)
+elide the creation
+of the temporary object, 
 effectively only calling the destructor of the source
 object at the end of the expression evaluation. If this optimization is done, 
 the temporary returned by `reloc` is not materialized.
 
 This means that `reloc obj;` has the following behavior:
 
 - if `obj` is not an *unowned parameter* then:
-    - If `obj` has a relocation or move constructor,
```

Comment:
I didn't understand this part. Is this related to #15 ?

Reply:
Yes, the goal was to tackle this issue.